src/Microsoft.AspNetCore.SignalR/HubEndPoint.cs

```diff
@@ -232,7 +232,7 @@ private void DiscoverHubMethods()
         private static bool IsHubMethod(MethodInfo m)
         {
             // TODO: Add more checks
-            return m.IsPublic;
+            return m.IsPublic && typeof(Hub<IClientProxy>) != m.GetBaseDefinition().DeclaringType;
```

Comment:
Old SignalR did it through the IHub interface https://github.com/SignalR/SignalR/blob/1fba14fa3437e24c204dfaf8a18db3fce8acad3c/src/Microsoft.AspNet.SignalR.Core/Hubs/ReflectionHelper.cs#L24